### ISSUE TITLE
Reports: Loss curves and images generated

### ISSUE THREAD

**Issue by spencerwooo** [spencerwooo/advsteg-pytorch#1]
### Cover images

![cover](https://user-images.githubusercontent.com/32114380/205899913-9c2f75c4-3d57-4608-85b9-bd99efa6bc9b.png)

### Loss curves on first 100 epochs

Trained on 10% of the CelebA dataset for 100 epochs.

```bash
python train.py --cuda --batch-size=128 --epochs=100 --fraction=0.1
```

![epoch100](https://user-images.githubusercontent.com/32114380/206093231-58de67a5-9566-40b0-beb3-fa4bb9e1c9b4.png)

Generated steganographic images:

![output100](https://user-images.githubusercontent.com/32114380/206091177-0508f2a4-bec3-4386-8e50-91239b79536b.png)

### Loss curves on 500 epochs

Trained on the entire CelebA dataset for 500 epochs.

```bash
python train.py --cuda --batch-size=256 --epochs=500
```

![epoch500](https://user-images.githubusercontent.com/32114380/206092808-66e4494c-60e8-46b8-84da-63bf63b3d5f5.png)

Generated steganographic images:

![output](https://user-images.githubusercontent.com/32114380/206091199-f01d6d13-54ae-435c-9c3e-2907fe0481cf.png)
